serialize.go

```diff
@@ -124,21 +130,46 @@ func (j *Jar) allPersistentEntries() []entry {
 	return entries
 }
 
-// lockFileName returns the name of the lock file associated with
-// the given path.
-func lockFileName(path string) string {
-	return path + ".lock"
-}
+const maxRetryDuration = 2 * time.Second
+
+var re = regexp.MustCompile(`[^a-z0-9A-Z]*`)
```

Comment:
I'd name this something more descriptive given it's a top level variable (even though private).